extensions/amp-video/0.1/amp-video.js

```diff
@@ -120,6 +156,8 @@ class AmpVideo extends AMP.BaseElement {
           'No "poster" attribute has been provided for amp-video.');
     }
 
+    this.isPrerenderAllowed_ = this.hasAnyCachedSources_();
```

Comment:
Nit, this won't affect prerendering of the video poster right, which is currently always prerendered? Maybe we should add a comment?

Reply:
done